docs/static/releasenotes.asciidoc

```diff
@@ -5,9 +5,51 @@ coming[8.0.0]
 
 This section summarizes the changes in the following releases:
 
+* <<logstash-8-0-0-beta1,Logstash 8.0.0-beta1>>
 * <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+[[logstash-8-0-0-beta1]]
+=== Logstash 8.0.0-beta1 Release Notes
+
+[[nanosecond-precision]]
+As processing times speed up, millisecond granularity is not always enough. Inbound data increasingly has sub-millisecond granularity timestamps.
+The pull request https://github.com/elastic/logstash/pull/12797[#12797] allows the internal mechanisms of Logstash that hold moment-in-time data - such as the Logstash Event, the Persistent Queue, the Dead Letter Queue and JSON encoding/decoding - to have nanosecond granularity.
+
+Timestamp precision is limited to the JVM and Platform's available granularity, which in many cases is microseconds.
+
+This change also grants users access to https://docs.oracle.com/javase/8/docs/api/java/time/format/DateTimeFormatter.html#patterns[Java time's improved formatters], which include support fort ISO quarters, week-of-month, and a variety of timezone/offset-related format substitutions. For example:
+
+[source,json]
+--------------------------------------------------------------------------------
+filter {
+  mutate {
+    add_field => {"nanos" => "Nanos: %{{n}}" }
+  }
+}
+--------------------------------------------------------------------------------
+
+Results in the following event:
+
+[source,json]
+--------------------------------------------------------------------------------
+{
+    "@timestamp" => 2021-10-31T22:32:34.747968Z,
+          "host" => "logstash.lan",
+         "nanos" => "Nanos: 747968000",
+       "message" => "test",
+          "type" => "stdin",
+      "@version" => "1"
+}
+--------------------------------------------------------------------------------
+
+NOTE: These prs below are also in 7.15/7.16, we can delete these entries before v8.0.0-beta1 release
+* Secure API https://github.com/elastic/logstash/pull/13308[#13308]
+* geoip integrate air-gapped bootstrap script https://github.com/elastic/logstash/pull/13104[#13104]
+* Add deprecation warnings for JAVA_HOME/… https://github.com/elastic/logstash/pull/13365[#13365]
+* Fixes to build and run Logstah on JDK 17 https://github.com/elastic/logstash/pull/13306[#13306]
+
```

Comment:
I think we're missing some content here around dropping support for java 8. I can work with @karenzone on the language here